osmdroid-android/src/main/java/org/osmdroid/views/overlay/Marker.java

```diff
@@ -571,7 +571,8 @@ protected void drawAt(final Canvas pCanvas, final int pX, final int pY, final fl
             pCanvas.save();
             pCanvas.rotate(pOrientation, pX, pY);
         }
-        if (mIcon instanceof BitmapDrawable) { // optimization 3: (about 15% faster)
+        /*
```

Comment:
I would add a single comment like "Unfortunate optimization with displayed size side effects"

Reply:
Sure. But I don't know how to add that change in the PR :-(

Reply:
Found!